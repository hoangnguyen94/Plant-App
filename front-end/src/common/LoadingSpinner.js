import React from "react";
import "./LoadingSpinner.css";

/** Loading message used by components that fetch API data. */

const LoadingSpinner = () =>
{
    return (
        <div className="LoadingSpinner">
            Loading ...
        </div>
    );
}

export default LoadingSpinner;