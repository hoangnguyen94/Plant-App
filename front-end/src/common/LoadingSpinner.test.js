import React from 'react';
import { render } from '@testing-library/react';
import LoadingSpinner from './LoadingSpinner';

describe('LoadingSpinner component', () => {
  test('renders the loading spinner component', () => {
    const { container } = render(<LoadingSpinner />);
    
    expect(container.firstChild).toMatchSnapshot();
  });
});
