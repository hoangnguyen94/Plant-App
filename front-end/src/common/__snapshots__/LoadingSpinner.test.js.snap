// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner component renders the loading spinner component 1`] = `
<div
  class="LoadingSpinner"
>
  Loading ...
</div>
`;
